### PR TITLE
{bio}[GCCcore/8.3.0] canu v1.9

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

Requires:
* [x] gnuplot-5.2.8-GCCcore-8.3.0.eb (#9737)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3d49fcbcfb418126fa2560ef3e4279b7 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3f6e14fafa554d0fc8cfa0f22b721a5b for a full test report.

**Comment by migueldiascosta** [comment]
closing and reopening to trigger tests

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/90b16e0f36b3a31ef8d803281a493041 for a full test report.

File: easybuild/easyconfigs/c/canu/canu-1.9-GCCcore-8.3.0-Perl-5.30.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'canu'
+version = '1.9'
+versionsuffix = '-Perl-%(perlver)s'
```
Comment (Micket) [inline easybuild/easyconfigs/c/canu/canu-1.9-GCCcore-8.3.0-Perl-5.30.0.eb]: I think we can/should drop the Perl suffix; there is just one perl version per toolchain release (this seems to be the case in the packages for GCCcore 8.3.0 so far). However; Java can and does vary, so '-Java-%(javaver)s' instead.

Comment (boegel) [reply easybuild/easyconfigs/c/canu/canu-1.9-GCCcore-8.3.0-Perl-5.30.0.eb]: +1 on this

Comment (branfosj) [reply easybuild/easyconfigs/c/canu/canu-1.9-GCCcore-8.3.0-Perl-5.30.0.eb]: done

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bber0501u03b.bb2.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ada0a3e7347ce00940fd0c784dd7b63f for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/70ffa50b5c3430636e1668778e2ddef8 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @branfosj!

**Pull request merged**